### ISSUE TITLE
Image mapping value problem

### ISSUE THREAD

**Issue by Morgan-rgb-del** [lamm-mit/FieldPredictorGAN#1]
Your method is very inspiring to me, I am also working on the corresponding GAN for field estimation, but I would like to ask that is there a part of your code that remaps the image back to the value, the color represents the corresponding value, but is it standard to calculate the value back this way?

**Comment by zhenzey** [comment]
Yes, it is standard to calculate the value back from color. And the visualization tool of Abaqus sometimes does not output the exact RGB value of a color. So it is better to output element wise strain/stress values and then do the plotting/visualization using other tools like Python.
